generics/services/samiksha-service.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * name : samiksha-service.js
+ * author : Rakesh Kumar
+ * Date : 4-June-2020
+ * Description : All samiksha service related api call.
+ */
+
+//dependencies
+
+let urlPrefix =
```

Comment:
@rakeshSgr - This should be a constant and rename it to samikshaServiceBaseURL

Reply:
resolved